sha1-checked/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "sha1-checked"
+version = "0.11.0-pre.3"
+description = "SHA-1 hash function with collision detection"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+edition = "2021"
+documentation = "https://docs.rs/sha1-checked"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["crypto", "sha1", "hash", "digest"]
+categories = ["cryptography", "no-std"]
+rust-version = "1.72"
+
+[dependencies]
+digest = "=0.11.0-pre.8"
+sha1 = { version = "=0.11.0-pre.3", features = [], path = "../sha1" }
```

Comment:
I don't think you need to depend on `sha1`. Simply copy the initialization constant.

The `sha1` changes can be reverted completely (outside of the doc addition).

Reply:
I am using `sha1::compress` as fallback when detection is turned off, that's why the dependency exists

Reply:
I think we simply can make collision detection mandatory for this implementation. If users selected this crate over `sha1`, it means they intentionally want to have collision deteciton.

Reply:
I personally would prefer this setup over having to write a helper function when needing to configure this at runtime, but I am okay either way.